trxer.go

```diff
@@ -22,17 +22,20 @@ import "io"
 var UPDATE_INTERVAL = 5
 var PORT = 6666
 
-var BYTE_BUFFER_SIZE = 8096 * 8
+var DEF_BUFFER_SIZE = 8096 * 8
+var MIN_BUFFER_SIZE = 4096
+var MAX_BUFFER_SIZE = 212992
```

Comment:
What? Where comes the limit in? A can send 4GB into the TCP stack write() system call, right?